src/ProviderPR/SFA.DAS.ProviderPR/Application/Queries/GetEASUserByEmail/GetEasUserByEmailQueryHandler.cs

```diff
@@ -0,0 +1,45 @@
+﻿using MediatR;
+using SFA.DAS.SharedOuterApi.Configuration;
+using SFA.DAS.SharedOuterApi.InnerApi.Requests;
+using SFA.DAS.SharedOuterApi.InnerApi.Requests.EmployerAccounts;
+using SFA.DAS.SharedOuterApi.InnerApi.Responses;
+using SFA.DAS.SharedOuterApi.InnerApi.Responses.EmployerAccounts;
+using SFA.DAS.SharedOuterApi.Interfaces;
+using System.Net;
+
+namespace SFA.DAS.ProviderPR.Application.Queries.GetEasUserByEmail;
+public class GetEasUserByEmailQueryHandler(IAccountsApiClient<AccountsConfiguration> apiClient) : IRequestHandler<GetEasUserByEmailQuery, GetEasUserByEmailQueryResult>
+{
+    public async Task<GetEasUserByEmailQueryResult> Handle(GetEasUserByEmailQuery request, CancellationToken cancellationToken)
+    {
+        var result = await apiClient.GetWithResponseCode<GetUserByEmailResponse>(new GetUserByEmailRequest(request.Email));
+
+        if (result.StatusCode != HttpStatusCode.OK)
```

Comment:
Should handle 404 specifically here and throw `InvalidOperationException` for all other non-successful response.